CMakeLists.txt

```diff
@@ -26,8 +26,13 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+option(CLIENT_SIMULATOR "Install the client simulator binary" OFF)
```

Comment:
BTW, you'll need to edit .travis.yml in order to test the client simulator's installation. Note for later: have CI tests for the client simulator.